sdk/resourcemanager/agrifood/armagrifood/CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
 # Release History
 
+## 0.7.0 (2022-08-23)
+### Breaking Changes
+
+- Function `*FarmBeatsModelsClient.Update` has been removed
```

Comment:
Can we add informational change log like:
- Sensor, Private endpoint & managedIdentity support added.
- Refer https://github.com/MicrosoftDocs/Azure-FarmBeats/tree/main/documentation for more details.

Reply:
Changed. Also add the breaking change info.

Reply:
removed private repo refer